pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -42,6 +42,8 @@ import (
 
 const (
 	launchTemplateNameFormat  = "Karpenter-%s-%s"
+	karpenterManagedTagKey    = "karpenter.sh/managed-by"
+	karpenterManagedTagVal    = "Karpenter"
```

Comment:
Feels a bit odd for this value to matter. Should we toss the cluster name in?

Reply:
ohhh yes, you're right anyways since this could pull in LTs from other Karpenter clusters in the same account and region